terminateTestProcesses.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+# Arg S1 : Unix current env.User
+
+# If Windows uses PowerShell
+if [ "$OSTYPE" = "cygwin" ]; then
+  echo "Windows machine, using powershell queries..."
+
+  # Match anything that is most likely a test job or process related
+  match_str="(CommandLine like '%java%jck%' or \
+             CommandLine like '%openjdkbinary%java%' or \
+             CommandLine like '%java%javatest%' or \
+             CommandLine like '%java%-Xfuture%' or \
+             CommandLine like '%X%vfb%' or \
+             CommandLine like '%rmid%' or \
+             CommandLine like '%rmiregistry%' or \
+             CommandLine like '%tnameserv%' or \
+             CommandLine like '%make.exe%')"
+
+  # Ignore Jenkins agent and grep cmd
+  ignore_str="not CommandLine like '%remoting.jar%' and \
+              not CommandLine like '%agent.jar%' and \
+              not CommandLine like '%grep%'"
+
+  count=`powershell -c "(Get-WmiObject Win32_Process -Filter {${ignore_str} and ${match_str}} | measure).count" | tr -d "\\\\r"`
```

Comment:
Not sure why this has changed from the `GetProcess` used in the previous version of this function but this seems to be filtering on `CommandLine` which isn't the object returned by `Get-WmiObject Win32_Process`. Am I missing something?
![image](https://user-images.githubusercontent.com/6487691/206180381-26458b95-ac48-495a-b143-e7a358bc825d.png)